packages/editor/src/components/post-preview-button/index.js

```diff
@@ -192,7 +192,7 @@ export class PostPreviewButton extends Component {
 					}
 				</span>
 				<DotTip tipId="core/editor.preview">
-					{ __( 'Click “Preview” to load a preview of this page, so you can make sure you’re happy with your blocks.' ) }
+					{ __( 'Once you\’ve drafted some content, use the “Preview” button to see how it will look to readers. When you\’re ready to add it to your site, use the “Publish” button.' ) }
```

Comment:
The `\`s here are unnecessary since we're using "smart" apostrophes.

```suggestion
					{ __( 'Once you’ve drafted some content, use the “Preview” button to see how it will look to readers. When you’re ready to add it to your site, use the “Publish” button.' ) }
```